src/components/lib/udp_stack/simple_udp_stack.c

```diff
@@ -13,6 +13,7 @@ static u32_t host_ip;
 static u16_t host_port;
 
 static struct ether_addr nic_mac;
+#if 1
```

Comment:
This as well.

Reply:
@WenyuanShao @gparmer We cannot remove these #define or #if (this also makes sense for the previous similar comments @WenyuanShao makde, I reply all of them here) for two reasons:1. We need the code work for both on Qemu and the hardware. 2. We have potential bugs and perf problems, we need code to enable us detect them quickly. Of course there could be better ways to do this, but we are limited by time. Based on these, I think we should not remove them. If you think the reasons don't make sense, then I'll remove them.